test/win32/test.rc

```diff
@@ -6,7 +6,7 @@
 //
 // Microsoft Visual C++ generated resource script.
 //
-#include "resource.h"
+#include <win32/resource.h>
 #include <winresrc.h>
```

Comment:
not strictly necessary, but then i assume that App Studio ended up rewriting some part of the code later on (i.e. stuff like `LANGUAGE` directive